docs/DeveloperGuide.md

```diff
@@ -23,9 +23,18 @@ The add medication command extends from the Command parent class and implements
 - assertionTest(MedicationManager, DailyMedicationManager) - Asserts that medicine has been added to both medication managers.
 - parseStringToValues(Arguments) - Parses string input for medicine quantity and dosage into double type.
 
-Given below is an example usage scenario and how the add command behaves at each step.
-- Step 1. To be added.
-- Step 2. To be added.
+Given below is an example usage scenario, along with the sequence diagram and how the add command behaves at each step:
+
+![AddCommand.png](images/AddCommand.png)
```

Comment:
I think that your sequence diagrams could include the activation bar and method returns are to be in dashed lines.

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/139/commits/7025f29908504b1e4031ab6bbe293fc91997f115. 
Thank you!